app/Providers/RouteServiceProvider.php

```diff
@@ -110,7 +110,7 @@ private function mapWebRoutes()
                 'namespace'  => 'Frontend',
                 'prefix'     => '',
                 'as'         => 'frontend.',
-                'middleware' => ['auth'],
+                'middleware' => (setting('security.require_email_verification', false) ? ['auth', 'verified'] : ['auth']),
```

Comment:
I think we should make this required always, or at least put this setting in a config file. We don't want a database call being made for setting up the routes. Maybe under `config/phpvms.php`, after `installed`, as

```php
'registration' => [
  'email_validation' => true,
],
```

Reply:
Laravel don't have a toggle, the developer should implement the `MustVerifyEmail` in the User model, and add the middleware to the routes.

If we make this always required, it will force every pilot to verify their e-mails on this phpvms update

Reply:
Every update? Why would that be the case?

Reply:
Not, just this update

Reply:
As part of the migration, can you set the verified timestamp to now? That should avoid it